test/arrays/views/assignSlice.chpl

```diff
@@ -0,0 +1,19 @@
+const D = {1..9, 1..9};
+var A: [D] real;
+
+forall (i,j) in D do
+  A[i,j] = i + j/10.0;
+
+A[5..6, 7..8] = 0.0;
+
+writeln(A, "\n");
+
+forall a in A[5..6, 7..8] do
+  a = 1.0;
+
+writeln(A, "\n");
+
+forall a in A[5..6, 7..8] do
```

Comment:
No strong reason... it was just a simple stepping stone test to get things bootstrapped and up and running.  Once they were, I relied on pre-existing tests for more complete coverage.  Put another way, I wasn't trying to write tests that exercised every possible way of using views, hoping that we had enough coverage in the test system that I wouldn't need to put a lot of effort in here.  (That said, given that there are things I know are wrong that the test system didn't catch, that isn't the best assumption).